packages/datadog-instrumentations/src/express.js

```diff
@@ -3,8 +3,18 @@
 const { createWrapRouterMethod } = require('./router')
 const shimmer = require('../../datadog-shimmer')
 const { addHook, channel, tracingChannel } = require('./helpers/instrument')
+const {
+  setRouterMountPath,
+  joinPath,
+  getLayerMatchers,
+  normalizeMethodName,
+  markAppMounted
+} = require('./helpers/router-state')
+
+const METHODS = [...require('http').METHODS.map(v => v.toLowerCase()), 'all']
 
 const handleChannel = channel('apm:express:request:handle')
+const routeAddedChannel = channel('apm:express:add:route')
```

Comment:
nit: makes more sense in this syntax
```suggestion
const routeAddedChannel = channel('apm:express:route:add')
```

Reply:
I think this is because the one you are suggesting is already in use for not exactly the same purpose. Maybe we can create some tag name more specific, I dunno, `apm:express:route:defined` maybe?